app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt

```diff
@@ -165,7 +165,9 @@ class MenuDrawerFragment : MenuFoldersFragment(), MailboxListFragment {
         }
 
         appVersionName.setOnClickListener {
-            (requireActivity() as MainActivity).onEasterEggConfettiClicked(matomoValue = "MenuDrawer")
+            (activity as? MainActivity)?.binding?.easterEggConfettiContainer?.let {
+                ConfettiUtils.onEasterEggConfettiClicked(container = it, matomoValue = "MenuDrawer")
+            }
```

Comment:
We should do something more reusable than relying on the MainActivity and aime for weaker coupling